plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/IcebergRestCatalogConfig.java

```diff
@@ -123,4 +125,17 @@ public IcebergRestCatalogConfig setVendedCredentialsEnabled(boolean vendedCreden
         this.vendedCredentialsEnabled = vendedCredentialsEnabled;
         return this;
     }
+
+    @Config("iceberg.rest-catalog.namespace-separator")
+    @ConfigDescription("Sets namespace separator character.")
+    public IcebergRestCatalogConfig setNamespaceSeparator(String separator)
+    {
+        this.namespaceSeparator = Optional.ofNullable(separator);
+        return this;
+    }
+
+    public Optional<@Pattern(regexp = "^\\.$", message = "The value must be a single period '.'") String> getNamespaceSeparator()
```

Comment:
Move `getNamespaceSeparator` above `setNamespaceSeparator`. We usually put getter method first.

Reply:
done